### DISCUSSION TITLE
How to include images in extensions such as interfaces, layouts or modules

### DISCUSSION THREAD

**Discussion by alexbrowng** [directus/directus#7314] [Archive]
I have created an interface with different inputs for a json field to store the translations with the following format:

`{"en": "Lorem ipsum", "es": "Lorem ipsum"}`

In each of the inputs I want to indicate a language flag but I can't include the image after compiling the extension. 

Does anyone know how to do it? 
Thanks

**Comment by rijkvanzanten** [comment]
@nickrum I'm assuming this would require a (custom) Rollup plugin in the extension builder setup?

**Reply by nickrum** [reply]
Yes, this would be possible by using something like `@rollup/plugin-image`.

**Reply by alexbrowng** [reply]
Can you please provide an example?
